.github/workflows/vitess_tester_vtgate.yml

```diff
@@ -112,7 +112,7 @@ jobs:
         go install github.com/vitessio/go-junit-report@HEAD
         
         # install vitess tester
-        go install github.com/vitessio/vitess-tester@eb953122baba163ed8ccaa6642458ee984f5d7e4
+        go install github.com/vitessio/vitess-tester@e9f56a7
```

Comment:
Would recommend to keep using full shas here in case somewhere in the future it accidentally starts colliding and you get very weird random failures.

Reply:
👍 
I'm updating `vitess-tester` to make it possible to run these tests. https://github.com/vitessio/vitess-tester/pull/19
Once that PR is merged, I'll update this sha to the full version.